src/status_im/subs.cljs

```diff
@@ -1282,15 +1285,33 @@
 (re-frame/reg-sub
  :chats/photo-path
  :<- [:contacts/contacts]
+ :<- [:profile/multiaccount]
+ :<- [:mediaserver/port]
+ (fn [[contacts multiaccount port] [_ id identicon]]
+   (let [contact (or (get contacts id)
+                     (when (= id (:public-key multiaccount))
+                       multiaccount))]
+     (if-not (nil? contact)
+       (multiaccounts/displayed-photo contact)
+       (if @config/new-ui-enabled?
```

Comment:
why only for new design?

Reply:
oh sorry, i think i misunderstood "Identity ring should be toggled, as we don't want to show it in the old UI, only on the new UI", when new-ui not enabled, we should also use image server but no ring, thanks for point out! @flexsurfer